src/components/markdown_editor/markdown_editor.tsx

```diff
@@ -185,7 +223,11 @@ export const EuiMarkdownEditor = forwardRef<
       [editorId, toolbarPlugins.map(({ name }) => name).join(',')]
     );
 
-    const classes = classNames('euiMarkdownEditor', className);
+    const classes = classNames(
+      'euiMarkdownEditor',
+      height === 'full' && 'euiMarkdownEditor--fullHeight',
```

Comment:
It's easier to read this as:
```suggestion
     {'euiMarkdownEditor--fullHeight': height === 'full'},
```